contribs/gmf/src/controllers/desktop.scss

```diff
@@ -247,7 +246,7 @@ gmf-backgroundlayerselector {
 $theme-selector-columns: 2;
 .gmf-theme-selector li {
   float: left;
-  width: calc((100% - $theme-selector-columns * 2 * $half-app-margin) / $theme-selector-columns);
+  width: calc((100% - #{$theme-selector-columns} * 2 * #{$half-app-margin}) / #{$theme-selector-columns});
```

Comment:
What's this syntax?

Reply:
The equivalent to calc(100% - ~"@{theme-selector-columns} * 2 ...") in less

So a manner to use `calc` with sass variables